gateway_code/control_nodes/cn_iotlab/__init__.py

```diff
@@ -190,7 +190,7 @@ def _wait_control_node_ready():
         So wait 2 seconds to be safe.  """
         time.sleep(2)
 
-    @staticmethod
-    def status():
+    @logger_call("Control Node : status")
```

Comment:
Yeah, this looks fine, but this is not related to standalone M3 control node IMO. Put that un another PR, with other small fixes

Reply:
Well, it's actually breaking the linter, because the status method should be static (we don't use self in it). either ignore it (add appropriate pylint: ignore),